### PR TITLE
Migrate to Fluentd v0.14 API

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by y-ken** [comment]
Thank you

File: lib/fluent/plugin/in_mysql_query.rb [hunk 1]
```diff
@@ -26,30 +29,24 @@ def configure(conf)
     end
 
     def start
-      @thread = Thread.new(&method(:run))
-    end
-
-    def shutdown
-      Thread.kill(@thread)
+      super
+      timer_execute(:in_mysql_query, @interval, &method(:on_timer))
```
Comment (y-ken) [inline lib/fluent/plugin/in_mysql_query.rb]: nice improvement

**Comment by y-ken** [comment]
To release final version for fluentd v0.12.x, some of code has updated.
Would you please resolve conflicts?

**Comment by okkez** [comment]
@y-ken Resolved conflicts.

**Comment by okkez** [comment]
@y-ken Any progress?

**Comment by okkez** [comment]
@y-ken ping?

**Comment by okkez** [comment]
@y-ken FYI: [Fluentd v0\.14\.22 has been released \| Fluentd](https://www.fluentd.org/blog/fluentd-v0.14.22-has-been-released)

It's time to support Fluentd v0.14!

**Comment by y-ken** [comment]
It's the time to release v1.0.0
Would you please update version to v1.0.0?

**Comment by okkez** [comment]
@y-ken Sorry, I missed your comment. I've updated version to v1.0.0.

**Comment by y-ken** [comment]
Is it the time to merge this p-r?
After that, I'll release new gem soon.

**Comment by okkez** [comment]
Yes, it's time to merge this PR!

**Comment by y-ken** [comment]
I've released v1.0.0 now!
https://rubygems.org/gems/fluent-plugin-mysql-query/versions/1.0.0

**Pull request merged**